modules/apiserversource-kn.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assemblies:
+//
+// serverless/knative_eventing/serverless-kn-source.adoc
+
+[id="apiserversource-kn_context"]
+= Using the ApiServerSource with the Knative CLI (`kn`)
+
+This guide describes the steps required to create, manage and delete an ApiServerSource using `kn` commands.
+
+.Prerequisites
+
+* You will need to have a Knative Serving and Eventing installation.
+* You will need to have created the `default` broker in the same namespace that the ApiServerSource will be installed in.
+* You will need to have the `kn` CLI installed.
+
+.Procedure
+
+. Create a service account, role and role binding for the ApiServerSource.
++
+You can do this by creating a file named `authentication.yaml` and copying the following sample code into it:
++
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: events-sa
+  namespace: default <1>
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: event-watcher
+  namespace: default <1>
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - events
+    verbs:
+      - get
+      - list
+      - watch
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: k8s-ra-event-watcher
+  namespace: default <1>
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: event-watcher
+subjects:
+  - kind: ServiceAccount
+    name: events-sa
+    namespace: default <1>
+----
++
+<1> Adjust this namespace to the namespace that you have selected for installing ApiServerSource.
++
+[NOTE]
+====
+If you want to re-use an existing service account with the appropriate permissions, you must modify the `authentication.yaml` for that service account.
+====
++
+Create the service account, cluster role and cluster binding by entering the following command:
++
+----
+$ oc apply --filename authentication.yaml
+----
+
+. Create an ApiServerSource event source by entering the following command:
++
+----
+$ kn source apiserver create testevents --sink broker:default --resource "event:v1" --service-account events-sa --mode Resource
+----
+
+. To check that the ApiServerSource is set up correctly, create a Knative service that dumps incoming messages to its log by entering the following command:
++
+----
+$ kn service create event-display --image quay.io/openshift-knative/knative-eventing-sources-event-display
```

Comment:
Should use a specific tag for the current release:

kn service create event-display --image quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2